capstone/scripts/count_chars.py

```diff
@@ -0,0 +1,62 @@
+from collections import Counter
+
+from celery import shared_task
+from capdb.models import CaseXML
+import json
+from os import mkdir
+from scripts.helpers import extract_casebody
+
+def count_chars_in_all_cases(path):
+    """
+        iterate through all volumes, call celery task for each volume
+    """
+    query = CaseXML.objects.all()
+
+    # if not updating existing, then only launch jobs for volumes with unindexed cases:
+
+    print("😃😀😃😃😃😀😃😃\n\nThis will dump a json file for each case in the {} directory, which ".format(path), end="")
+    try:
+        mkdir(path)
+        print("I just created.\n\n😃😀😃😃😃😀😃😃")
+    except FileExistsError:
+        print("already exists.\nNote: this will overwrite existing data, but unless the cases or script have changed, it should be the same data.\n\n😃😀😃😃😃😀😃😃")
+
+    # launch a job for each volume:
+    for cmd_id in query.values_list('pk', flat=True):
+        count_case_chars.delay(cmd_id, path, write_to_disk=True)
```

Comment:
I think for celery to find tasks on prod they have to be listed in `config/celery.py`